src/lib/functions.js

```diff
@@ -3,6 +3,14 @@
  */
 
 /**
+ * Check to see this is a node environment.
+ * @type {Boolean}
+ */
+/* global global */
+export const isNode = typeof global !== 'undefined' &&
```

Comment:
Is this an accurate check?

Could we instead get away with something dead simple like: `document && document.querySelector`

Reply:
@cfebs that test didn't seem to work, not sure why.
Could you approve if it looks alright to you?